recorded_future/komand_recorded_future/actions/lookup_domain/action.py

```diff
@@ -15,7 +15,8 @@ def __init__(self):
 
     def run(self, params={}):
         try:
-            domain = params.get(Input.DOMAIN).lstrip('https://').split('/')[0]
+            original_domain = params.get(Input.DOMAIN)
+            domain = original_domain.replace('https://','').split('/')[0]
```

Comment:
Is https required? If not, a URL starting with http will cause `domain` to become `http:`

Reply:
Make sure we test with other TLDs ending in an `s` - if that's the only character this effects e.g. `.as` to be sure.

https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains